docs/AMBASSADOR.md

```diff
@@ -15,7 +37,7 @@ Obviously the actual pod names will likely be different when you run this. Once
 set up port forwarding:
 
 ```
-$ kubectl --namespace=ambassador port-forward ambassador-8db4cb58d-gsmjj 8877 8877
+$ kubectl --namespace=ambassador-edge port-forward ambassador-8db4cb58d-gsmjj 8877 8877
```

Comment:
2 things:

1. I just learned you can port-forward to a *deployment* and the deployment then picks the pod. That means you don't need to first list the pods, etc. just do a `kubectl port-forward deployment/ambassador ...`
2. You're forwarding the same port twice.

Reply:
oh that's nice